recipes/sbp/all/conanfile.py

```diff
@@ -0,0 +1,70 @@
+import os
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class SbpConan(ConanFile):
+    name = "sbp"
+    license = "MIT"
+    homepage = "https://github.com/swift-nav/libsbp"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Swift Binary Protocol client library"
+    topics = ("gnss",)
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    generators = "cmake"
+    exports_sources = "CMakeLists.txt", "c"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+        if self.settings.os == "Windows" and self.options.shared:
+            raise ConanInvalidConfiguration("Windows shared builds are not supported right now, see issue https://github.com/swift-nav/libsbp/issues/1062")
+
+    def source(self):
+        data = self.conan_data["sources"][self.version]
+
+        source = {"url": data["url"]["source"], "sha256": data["sha256"]["source"]}
+        tools.get(**source, strip_root=True, destination=self._source_subfolder)
+
+        cmake = {"url": data["url"]["cmake"], "sha256": data["sha256"]["cmake"]}
+        tools.get(**cmake, strip_root=True, destination=os.path.join(self._source_subfolder, "c", "cmake", "common"))
```

Comment:
```suggestion
        tools.get(**self.conan_data["sources"][self.version]["source"),
                  destination=self._source_subfolder, strip_root=True)
        tools.get(**self.conan_data["sources"][self.version]["cmake"),
                  destination=os.path.join(self._source_subfolder, "c", "cmake", "common"), strip_root=True)
```

Reply:
per above; I used the same approach as https://github.com/conan-io/conan-center-index/pull/6565#discussion_r678910840